src/test/unit/engine/LinkTest.cpp

```diff
@@ -247,7 +247,7 @@ TEST(LinkTest, DelayedLink) {
 
 
 
-TEST(LinkTest, DelayedLinkSerialization) {
+TEST(LinkTest, DISABLED_DelayedLinkSerialization) {
```

Comment:
fix/enable this testcase (I don't see any obvious merge conflict to cause this)

Reply:
> fix/enable this testcase (I don't see any obvious merge conflict to cause this)

I don't understand the cause either.  After the deserialization, the output was all zeros.  I will work on this.  I don't like checking in known broken things.

Reply:
I'm wondering if the templates or macros can mess some things up? As on the first sight there's no reason for this to break :/

Reply:
..so it was in a debug print